contracts-periphery/Makefile

```diff
@@ -6,8 +7,11 @@ else
   INSTALL_CMD := forge install
 endif
 
+forkBlock = 14800081
+
 build :; forge build
 clean :; forge clean
 install :; $(INSTALL_CMD)
-snapshot :; forge snapshot --match-path test/UmbraBatchSend.gas.t.sol
-test :; forge test
\ No newline at end of file
+snapshot :; forge snapshot -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path \*.gas.t.sol
+test :; forge test --match-path test/UmbraBatchSend.t.sol && make test-hook && forge test --match-path test/gas/UmbraBatchSend.gas.t.sol
+test-hook :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path test/UniswapWithdrawHook.fork.1.t.sol
```

Comment:
I might have made it worse? But check it out https://github.com/ScopeLift/umbra-protocol/pull/341/commits/63bf010e98444a791515d4318d060e4ad53dcf7b

Reply:
Latest update: https://github.com/ScopeLift/umbra-protocol/pull/341/commits/1f2e316d61a78f415f81c382983e1de64a334795

Reply:
This looks great, thanks! My only nit would be to use e.g. `test-gas-batch-send`instead of `test-batch-send-gas` as the command name, that way when sorted alphabetically all the gas commands are next to each other. Not a big deal though, feel free to update if you want + mark this one resolved

Reply:
I like the suggestion and changed it accordingly!